Orange/widgets/data/owselectrows.py

```diff
@@ -73,7 +73,7 @@ def __init__(self):
         self.cond_list = QtGui.QTableWidget(
             box, showGrid=False, selectionMode=QtGui.QTableWidget.NoSelection)
         box.layout().addWidget(self.cond_list)
-        self.cond_list.setColumnCount(3)
+        self.cond_list.setColumnCount(4)
```

Comment:
If you look a few lines below, the resize mode for all columns is set to `QHeaderView.Stretch`. Use `QHeaderView.setResizeMode()` to set that to `QHeaderView.ResizeToContents` or `QHeaderView.Fixed` (with provided fixed size) for the newly last column. No need for the small button to take ¹/₄ of space.